.github/workflows/bazelBuildAndTestTcp.yml

```diff
@@ -53,9 +53,9 @@ jobs:
                    -v "$(pwd)":"/opt/src/mlir-tcp" \
                    -v "${HOME}/.cache/bazel":"${HOME}/.cache/bazel" \
                    mlir-tcp:ci \
-                   find . -type f -name "*.cpp" -o -name "*.h" | xargs clang-format -i
+                   find . -type f -name "*.cpp" -a ! -name "*execute_test.template.cpp" -o -name "*.h" | xargs clang-format -i
         if [ -n "$(git status --porcelain)" ]; then
-          echo "Please run 'find . -type f -name "*.cpp" -o -name "*.h" | xargs clang-format -i' and commit changes."
+          echo "Please run 'find . -type f -name "*.cpp" -a ! -name "*execute_test.template.cpp" -o -name "*.h" | xargs clang-format -i' and commit changes."
```

Comment:
Prefer using a `.clang-format-ignore` file at the project root to track ignored files, to avoid this command from blowing up.